base/strings/util.jl

```diff
@@ -135,15 +135,16 @@ function chomp(s::String)
 end
 
 """
-    lstrip(str::AbstractString[, chars])
+    lstrip([pred,] str::AbstractString[, chars])
```

Comment:
It's not allowed to supply both `pred` and `chars` so this should be two separate signatures, something like this:
```jl
lstrip([pred=isspace,] str::AbstractString)
lstrip(str::AbstractString[, chars::Base.Chars])
```